<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QTableView" name="playlistView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>401</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="stopButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>24</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Icons/stop.png</normaloff>Icons/stop.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="prevButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>24</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Icons/prev.png</normaloff>Icons/prev.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="playButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>24</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Icons/play.png</normaloff>Icons/play.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>24</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Icons/next.png</normaloff>Icons/next.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="muteButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>24</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Icons/volume.png</normaloff>Icons/volume.png</iconset>
   </property>
  </widget>
  <widget class="QSlider" name="volumeSlider">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="durationSlider">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>401</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="styleComboBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
